cmd/operator-sdk/bundle/create_test.go

```diff
@@ -0,0 +1,221 @@
+      --default-channel stable
+
+Assuming your operator has the same name as your repo directory and the only
+channel is 'stable', the above command can be abbreviated to:
+
+  $ operator-sdk bundle create quay.io/example/test-operator:v0.1.0
+
+The following invocation will generate test-operator bundle metadata and a
+bundle.Dockerfile for your latest operator version without building the image:
+
+  $ operator-sdk bundle create \
+      --generate-only \
+      --package test-operator \
+      --channels beta \
+      --default-channel beta
+`))
+			flag := cmd.Flags().Lookup("directory")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("d"))
+			Expect(flag.Usage).To(Equal("The directory where bundle manifests are located, " +
+				"ex. <project-root>/deploy/olm-catalog/test-operator/manifests"))
+
+			flag = cmd.Flags().Lookup("output-dir")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("o"))
+			Expect(flag.Usage).To(Equal("Optional output directory for operator manifests"))
+
+			flag = cmd.Flags().Lookup("tag")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("t"))
+			Expect(flag.Usage).To(Equal("The path of a registry to pull from, image name and " +
+				"its tag that present the bundle image (e.g. quay.io/test/test-operator:v0.1.0)"))
+
+			flag = cmd.Flags().Lookup("package")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("p"))
+			Expect(flag.Usage).To(Equal("The name of the package that bundle image belongs to. " +
+				"Set if package name differs from project name"))
+
+			flag = cmd.Flags().Lookup("channels")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("c"))
+			Expect(flag.DefValue).To(Equal("stable"))
+			Expect(flag.Usage).To(Equal("The comma-separated list of channels that bundle image belongs to"))
+
+			flag = cmd.Flags().Lookup("generate-only")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("g"))
+			Expect(flag.DefValue).To(Equal("false"))
+			Expect(flag.Usage).To(Equal("Generate metadata/, manifests/ " +
+				"and a Dockerfile on disk without building the bundle image"))
+
+			flag = cmd.Flags().Lookup("overwrite")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.DefValue).To(Equal("false"))
+			Expect(flag.Usage).To(Equal("Overwrite bundle.Dockerfile, manifests and metadata dirs if they exist. " +
+				"If --output-dir is also set, the original files will not be overwritten"))
+
+			flag = cmd.Flags().Lookup("image-builder")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("b"))
+			Expect(flag.DefValue).To(Equal("docker"))
+			Expect(flag.Usage).To(Equal("Tool to build container images. One of: [docker, podman, buildah]"))
+
+			flag = cmd.Flags().Lookup("default-channel")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("e"))
+			Expect(flag.Usage).To(Equal("The default channel for the bundle image"))
+		})
+	})
+
+	Describe("Validate", func() {
```

Comment:
Why the validate is in the create_test.go and not in the validate_test.go?

Reply:
this is testing the argument validation method of the create command, not the bundle validate command